grunt/clean.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     images: 'dist/img',
     less: 'dist/less',
     licenses: 'dist/Licenses',
-    e2e: ['e2e/dist', 'e2e/_test-output', 'e2e/coverage'],
+    e2e: ['e2e/dist', 'e2e/_test-output', 'e2e/coverage', 'e2e/html', 'e2e/xml', 'e2e/chromedriver.exe'],
     shim: 'dist/dist',
-    bower: 'target/bower'
+    bower: 'target/bower',
+    webdriver: 'node_modules/protractor/node_modules/webdriver-manager/selenium'
```

Comment:
Why is this still being deleted? I don't understand.

Reply:
In the context of the continuous integration build this cleanup isn't needed as the repository is recloned for each build. My concern is what could happen when the 'e2e' task is run on development machines without this cleanup. On one day the chromedriver_2.34 binary could be downloaded by the 'run:webdriver_manager_update' task; on the next the chromedriver_2.35 could appear. The 'copy:webdriver' task would have to recognise the newer version before copying it beside the protractor configuration file. The simpler option seemed to be to delete the entire folder each time.

Reply:
Does it make sense to always use the latest version as soon as it comes out? Perhaps we should be sticking with a particular version until we can verify that a newer version does not introduce some unexpected behaviors?

Reply:
We can do that . To avoid storing the binary in the repository, a new task would be needed to download it. There are curl and wget plugins for grunt.

Reply:
Could you perhaps use npm to get the webdriver? E.g. https://www.npmjs.com/package/chromedriver. I suggest this because we can then use package.json to manage the version.

Reply:
Yes, that works. I will make that change.